src/lib/View/AppConfigCoreViewMainContentMapper.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\HybridPlatformUi\View;
+
+use eZ\Publish\Core\MVC\Symfony\View\View;
+use EzSystems\HybridPlatformUi\Components\App;
+use EzSystems\HybridPlatformUi\Components\MainContent;
+
+class AppConfigCoreViewMainContentMapper implements CoreViewMainContentMapper
+{
+    /**
+     * @var \EzSystems\HybridPlatformUi\Components\App
+     */
+    private $app;
+
+    public function __construct(App $app)
+    {
+        $this->app = $app;
+    }
+
+    /**
+     * @param \eZ\Publish\Core\MVC\Symfony\View\View $view
```

Comment:
The doc is supposed to be inherited from the parent without the need for `inheritDoc`.